common/internal_model/src/test/scala/weco/catalogue/internal_model/generators/ImageGenerators.scala

```diff
@@ -206,23 +206,23 @@ trait ImageGenerators
     } else {
       (1 to n).map(_ => randomVector(4096, maxR = 10.0f))
     }
-    val lshFeatures = if (similarFeatures) {
-      similarHashes(32, n)
+    val reducedFeatures = if (similarFeatures) {
+      similarVectors(1024, n)
```

Comment:
nit: not one for this PR, but it's hard to know what 1024/n are doing in this context (even with keyword arguments)